### PR TITLE
Updates ocky icky phobia to more modern lingo

### PR DESCRIPTION
**Pull request by Rex9001**
## About The Pull Request
As requested by the administrative team; ocky icky phobia has been updated with some all new lingo
Please give more ideas in the comments if you feel like something is missing!
Full list of words added:
		"amogus",
		"chat",
		"c*der",
                "coder",
		"coders",
		"erp",
		"joever",
		"kys",
		"lmao",
		"lol",
		"mapper",
		"muderbone",
		"murderboning",
		"powergame",
		"rp",
		"sprite",
		"spriter",
		"sus",
		"tide",
		"ur",
		"5head"
## Why It's Good For The Game
The ocky icky phobia is currently out of date with the hip and cool lingo the kids are using nowadays, this updates it for any admin feeling like granting a smite to an intrepid twitch user who is attempting to emote twitch emotes ic.

This improves the game because it allows for the currently very un-used ocky icky phobia smite to be more relevant in our current day and age.
## Changelog
:cl:
admin: Updates the admin smite "Ocky icky phobia" with some new words
/:cl:

### CODE REVIEW

**Comment by Rex9001** [comment]
Also made you fear the banhammer and coderbus plushie

**Comment by Jacquerel** [comment]
I don't think we can have "chat" in here that's a normal word
Like "let's chat" or "do you want to chat about it?"

**Comment by Rex9001** [comment]
> I don't think we can have "chat" in here that's a normal word Like "let's chat" or "do you want to chat about it?"

I added it because one of the headmins suggested it and the brain trauma is admin only, but I'd be fine changing it if you want

File: strings/phobia.json [hunk 1]
```diff
@@ -345,33 +345,52 @@
...
+		"lol",
+		"mapper",
 		"moderator",
 		"mods",
+		"muderbone",
+		"murderboning",
 		"ocky",
 		"ooc",
 		"owo",
 		"pwn",
```
Comment (DATA-xPUNGED) [inline strings/phobia.json:375]: who even uses this nowadays

File: strings/phobia.json [hunk 2]
```diff
@@ -345,33 +345,54 @@
 		"admeme",
 		"admin",
 		"ahelp",
+		"amongus",
```
Comment (Ghommie) [inline strings/phobia.json:348]: amogus

**Comment by DaCoolBoss** [comment]
Might as well add 'tide' to the greytide phobia while you're adding it to icky icky.

**Comment by githubuser4141** [comment]
Is 'coder' valid here? TG is set in the age of computers, there are several items in-game that refer to computers. It's totally valid to refer to coders/coding/code either in RP or plain text. In fact having a phobia to someone saying 'code' is way more OOC then saying it.

**Comment by Admiral-Gigaton** [comment]
> Is 'coder' valid here? TG is set in the age of computers, there are several items in-game that refer to computers. It's totally valid to refer to coders/coding/code either in RP or plain text. In fact having a phobia to someone saying 'code' is way more OOC then saying it.

when was the last time you or anyone else actually said coder, in one of the servers that values are pee, and wasnt talking about the coders? not many coder gimmicks, mainly because this isnt... goooooon... *shudder

**Comment by DaCoolBoss** [comment]
I don't think the TG setting even has coders. Circuits are close but I've never seen anyone refer to them as code. All the software is either hard printed on circuit boards by machines or uses artificial neurons (silicone).

**Comment by atlasle** [comment]
add round

**Comment by DATA-xPUNGED** [comment]
> Is 'coder' valid here? TG is set in the age of computers, there are several items in-game that refer to computers. It's totally valid to refer to coders/coding/code either in RP or plain text. In fact having a phobia to someone saying 'code' is way more OOC then saying it.

for something that is admin only and i have never seen literally anyone ever get, i think it's fine to exaggerate a little

**Comment by newfren** [comment]
do these trigger if they're contained within other words? e.g. would "turd" trigger "ur"?

**Comment by san7890** [comment]
that's probably out of scope for this pr

**Comment by newfren** [comment]
if it does trigger contained in other words, then ur and rp are probably a little too egregious to include

**Comment by Rex9001** [comment]
> add round 

Already on the list

File: strings/phobia.json (hunk 1 above)
Comment (Rex9001) [reply strings/phobia.json:377]: Lame people

**Comment by Rex9001** [comment]
> do these trigger if they're contained within other words? e.g. would "turd" trigger "ur"?

It shouldn't since "u" was already on the list and then you wouldn't also need to include words like "uwu"

**Comment by DaCoolBoss** [comment]
I can't check right now but from memory, the filter ignores trigger words as part of larger words like "ur" in "urn", with two exceptions. First, if an "s" is appended it assumes it's a plural form and blocks it (so the icky icky filter will block the word "us" as it's a plural of "u") or if some kind of speech filter separates the problem word ("yellow" is fine but "y-y-y-e-l-l-l-o-o-w-w" or "y-GLORF" isn't, because "y" becomes its own word). No idea how stuff like lizard lisps factor in.

**Comment by Rex9001** [comment]
This is a silly admin only smite, I'll keep it as is and if anyone is wondering I'm not gonna refactor all our phobias cause this one needed a small update

**Comment by Jacquerel** [comment]
yeah I'm honestly sorry I started this, the word list contents really aren't that big a deal
phobias being shit generally speaking is not in scope for this PR

**Comment by Exester509** [comment]
Aren't "lmao", "lol" and similar slang banned words? Unless you use circuits or some shit people aren't able to even say them

**Comment by DaCoolBoss** [comment]
Banned word list is server config, it's entirely possible a server runs the basic TG code but doesn't use it or has a different list of banned words.

**Comment by san7890** [comment]
> Aren't "lmao", "lol" and similar slang banned words? Unless you use circuits or some shit people aren't able to even say them

the phobia does the same `forced` stuff that allows you to bypass the word filters

**Comment by san7890** [comment]
anyways to not add words because the regex sucks is admitting defeat and succumbing to our fears instead of doing the right thing and fixing it in a task-specific PR

**Comment by Rex9001** [comment]
I havent seen too many regex issues with this pr, if there are any they are more likely to be stuff like "u" over stuff like "ur". Fixing our phobia regex is a task I'm not sure how to do which is why I havent felt comfortable claiming nor wishing to handle it here

**Pull request merged**